packages/mdc-select/mdc-select.scss

```diff
@@ -15,16 +15,14 @@
 //
 
 @import "./mixins";
+@import "./variables";
 @import "@material/animation/functions";
-@import "@material/animation/variables";
+// @import "@material/animation/variables";
```

Comment:
Probably don't need the commented out import.

Reply:
Ah! Will remove...